content/en/docs/setup/quickstart.md

```diff
@@ -8,100 +8,275 @@ menu:
 title: Quick Start
 toc: true
 type: docs
+description: >
```

Comment:
![image](https://github.com/kmesh-net/website/assets/85690076/2c0886e7-b35c-40c6-8e02-10bdc940e796)

Reply:
The user guide hasn't been written yet. Here's the quick start.

Reply:
![image](https://github.com/kmesh-net/website/assets/85690076/e5c6d119-cad9-427f-b0a4-84048a01b23a)

Reply:
https://deploy-preview-30--kmesh-net.netlify.app/en/docs/developer/
xx:v0.3.0  -> xx:latest

Reply:
With the exception of the `quickstart` docs, these are previous docs that haven't had time to be modified. We need to work together to modify the documents